backend/file-system/src/main/java/com/htec/filesystem/repository/FolderRepository.java

```diff
@@ -3,9 +3,12 @@
 import com.htec.filesystem.entity.FolderEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.List;
 import java.util.Optional;
 
 public interface FolderRepository extends JpaRepository<FolderEntity, Long> {
 
-    Optional<FolderEntity> findById(long id);
+    Optional<FolderEntity> findById(Long id);
```

Comment:
you missed this one

Reply:
I thought since function name is findById is okay for param name to be  "id". Changed to folderId